### PR TITLE
OSODCS-4373: Fixed procedure formatting in Deploying the cluster

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4..11+

Issue:
This PR addresses [osdocs-4373](https://issues.redhat.com/browse/OSDOCS-4373).

QE review:
This PR fixes broken formatting in Deploying the cluster. There are no changes to the technical accuracy/completeness of the procedure. Given the nature of the PR, QE review is not required. See [osdocs-4373](https://issues.redhat.com/browse/OSDOCS-4373) for complete details.

Link to docs preview:

Use case 1: Administrator runs the installation program without an `install.config.yaml` file.

- Installing a cluster quickly on AWS > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-default.html#installation-launching-installer_installing-aws-default)
- Installing a cluster quickly on GCP > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-default.html#installation-launching-installer_installing-gcp-default)
- Installing a cluster quickly on Azure > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-default.html#installation-launching-installer_installing-azure-default)
- Installing a cluster quickly on RHV > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-default.html#installation-launching-installer_installing-rhv-default)
- Installing a cluster on vSphere > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-launching-installer_installing-vsphere-installer-provisioned)
- Installing a cluster on VMC > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc.html#installation-launching-installer_installing-vmc)

Use case 2 (AWS and GCP): Administrator runs the installation program with an `install.config.yaml` file.

**AWS**

- Installing a cluster on AWS with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-launching-installer_installing-aws-customizations)
- Installing a cluster on AWS with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#installation-launching-installer_installing-aws-network-customizations)
- Installing a cluster on AWS in a restricted network > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.html#installation-launching-installer_installing-restricted-networks-aws-installer-provisioned)
- Installing a cluster on AWS into an existing VPC > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html#installation-launching-installer_installing-aws-vpc)
- Installing a private cluster on AWS > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private.html#installation-launching-installer_installing-aws-private)
- Installing a cluster on AWS into a government region > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-government-region.html#installation-launching-installer_installing-aws-government-region)
- Installing a cluster on AWS into a Secret or Top Secret Region > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-secret-region.html#installation-launching-installer_installing-aws-secret-region)
- Installing a cluster on AWS China > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-china.html#installation-launching-installer_installing-aws-china-region)

**GCP**

- Installing a cluster on GCP with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations.html#installation-launching-installer_installing-gcp-customizations)
- Installing a cluster on GCP with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-network-customizations.html#installation-launching-installer_installing-gcp-network-customizations)
- Installing a cluster on GCP in a restricted network > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.html#installation-launching-installer_installing-restricted-networks-gcp-installer-provisioned)
- Installing a cluster on GCP into an existing VPC > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-vpc.html#installation-launching-installer_installing-gcp-vpc)
- Installing a private cluster on GCP > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-private.html#installation-launching-installer_installing-gcp-private)

Use case 2 (all other platforms): Administrator runs the installation program with an `install.config.yaml` file.

**Alibaba**

- Installing a cluster quickly on Alibaba Cloud > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-default.html#installation-launching-installer_installing-alibaba-default)
- Installing a cluster on Alibaba Cloud with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-customizations.html#installation-launching-installer_installing-alibaba-customizations)
- Installing a cluster on Alibaba Cloud with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-network-customizations.html#installation-launching-installer_installing-alibaba-network-customizations)

**Azure**

- Installing a cluster on Azure with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-launching-installer_installing-azure-customizations)
- Installing a cluster on Azure with network customizations >[ Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-network-customizations.html#installation-launching-installer_installing-azure-network-customizations)
- Installing a cluster on Azure into an existing VNet > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-vnet.html#installation-launching-installer_installing-azure-vnet)
- Installing a private cluster on Azure > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-private.html#installation-launching-installer_installing-azure-private)
- Installing a cluster on Azure into a government region > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-government-region.html#installation-launching-installer_installing-azure-government-region)

**Azure Stack Hub**

- Installing a cluster on Azure Stack Hub with an installer-provisioned infrastructure > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-default.html#installation-launching-installer_installing-azure-stack-hub-default)
- Installing a cluster on Azure Stack Hub with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.html#installation-launching-installer_installing-azure-stack-hub-network-customizations)

**IBM Cloud VPC**

- Installing a cluster on IBM Cloud VPC with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.html#installation-launching-installer_installing-ibm-cloud-customizations)
- Installing a cluster on IBM Cloud VPC with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.html#installation-launching-installer_installing-ibm-cloud-network-customizations)

**Nutanix**

- Installing a cluster on Nutanix > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned.html#installation-launching-installer_installing-nutanix-installer-provisioned)

**OpenStack**

- Installing a cluster on OpenStack with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-launching-installer_installing-openstack-installer-custom)
- Installing a cluster on OpenStack with Kuryr > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-kuryr.html#installation-launching-installer_installing-openstack-installer-kuryr)
- Installing a cluster on OpenStack in a restricted network > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-restricted.html#installation-launching-installer_installing-openstack-installer-restricted)

**RHV**

- Installing a cluster on RHV with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations)

**vSphere**

- Installing a cluster on vSphere with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-launching-installer_installing-vsphere-installer-provisioned-customizations)
- Installing a cluster on vSphere with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#installation-launching-installer_installing-vsphere-installer-provisioned-network-customizations)
- Installing a cluster on vSphere in a restricted network > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.html#installation-launching-installer_installing-restricted-networks-installer-provisioned-vsphere)

**VMC**

- Installing a cluster on VMC with customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-customizations.html#installation-launching-installer_installing-vmc-customizations)
- Installing a cluster on VMC with network customizations > [Deploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations.html#installation-launching-installer_installing-vmc-network-customizations)
- Installing a cluster on VMC in a restricted network > D[eploying the cluster](https://51794--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-restricted-networks-vmc.html#installation-launching-installer_installing-restricted-networks-vmc)

### CODE REVIEW

**Comment by bscott-rh** [comment]
LGTM.

File: modules/installation-launching-installer.adoc [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // Module included in the following assemblies:
 //
-// installing/installing_alibaba/installing-alibaba-network-customizations.adoc
+// * installing/installing_alibaba/installing-alibaba-network-customizations.adoc
 // * installing/installing_alibaba/installing-alibaba-customizations.adoc
 // * installing/installing_alibaba/installing-alibaba-default.adoc
+// * installing/installing_alibaba/installing-alibaba-network-customizations.adoc
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply modules/installation-launching-installer.adoc:6]: Thanks. I will remove it.

**Pull request merged**